src/providers/outcome-behaviour-map/outcome-behaviour-map.ts

```diff
@@ -31,7 +37,7 @@ export class OutcomeBehaviourMapProvider {
     if (!field) {
       return false;
     }
-    if (field.display === 'A' && value !== null) {
+    if (field.display === 'A' && value) {
```

Comment:
The `VisibilityType` enum could be reused here.

Reply:
reused the enum.